amalgamate.py

```diff
@@ -1,31 +1,37 @@
 #!/usr/bin/env python
+"""
+Amalgamate PyNE's C++ library sources into a single header and source file.
+
+This script consolidates selected C++ components of the PyNE library into:
+    - pyne.h   : Self-contained header
+    - pyne.cpp : Self-contained source
 
-"""Amalgate pyne C++ library sources into a single source and a single header file.
-This makes the C++ portion of pyne more portable to other projects.
+Inspired by the JsonCpp amalgamation tool:
+    http://svn.code.sf.net/p/jsoncpp/code/trunk/jsoncpp/amalgamate.py
 
-Originally inspired by JsonCpp: http://svn.code.sf.net/p/jsoncpp/code/trunk/jsoncpp/amalgamate.py
+Usage:
+    python amalgamate.py [-s OUTPUT.cpp] [-i OUTPUT.h] [-f file1.h file2.cpp ...] [-o OUTPUT_DIR]
 """
+
 from __future__ import print_function, unicode_literals
 import os
+import subprocess
+from pathlib import Path
+from argparse import ArgumentParser
 import sys
 import io
-from argparse import ArgumentParser
-
-sys.path.append('pyne')
-import pyne_version as pv
 
-pv.write_cpp_header('src')
+# Configuration
+BASE_DIR = Path(__file__).resolve().parent
 
 CODE_EXTS = {".c", ".cpp", ".cxx", ".h", ".hpp", ".hxx"}
```

Comment:
(I know this is not something you added/changed, and it may defy my suggestion about scope creep, but you are modifying these lines anyway....)

Could we just use the following definition after we define the others?
```
CODE_EXTS = SOURCE_EXTS + HEADER_EXTS
```